package/MDAnalysis/analysis/nucleicacids.py

```diff
@@ -183,50 +302,234 @@ class WatsonCrickDist(NucPairDist):
 
     Attributes
     ----------
-    times: numpy.ndarray
-        Simulation times for analysis.
     results.pair_distances: numpy.ndarray
         2D array of Watson-Crick basepair distances. First dimension is
         simulation time, second dimension contains the pair distances for
         each each entry pair in strand1 and strand2.
 
         .. versionadded:: 2.4.0
 
+    results.distances: numpy.ndarray
+        stored in a 2d numpy array with first index selecting the Residue pair, 
+        and the second index selecting the frame number
+    times: numpy.ndarray
+            Simulation times for analysis.
 
     Raises
     ------
-    ValueError
-        If the residues given are not amino acids
+    DeprecationWarning
+    .. _Deprecation Warning
```

Comment:
Does this anchor work? Perhaps a trailing colon missing?

Reply:
neither anchor works, I'm having an issues building my docs locally that I asked about on discord. Once I sort that out I'll go over the docs again.